src/docs/about.md

```diff
@@ -0,0 +1,208 @@
+pipeline for your application, and is written in plan old JavaScript. The order in which operations happen is
+determined by this build file.
+
+You can think of broccoli-plugins much like a simple programming language, where the output of a function can be 
+passed as the input(s) to another function.
+
+E.g.:
+
+```js
+let js = babel('src');
+js = uglify(js);
+
+let css = sass('src/styles', 'site.scss');
+css = autoprefixer(css);
+
+const result = merge([js, css]);
+
+module.exports = result;
+```
+This could also be expressed as:
+```js
+module.exports = merge([
+    uglify(
+        babel('src')
+    ),
+    autoprefixer(
+        sass('src/styles', 'site.scss')
+    )
+]);
+```
+
+In the above, a `src` directory is passed to the `babel()` plugin (which will convert our new ES6 syntax into 
+ES5 that runs in the browser), the output of that is passed to the `uglify()` plugin which will minify our JS 
+into a smaller format. The output of `uglify()` will in turn be passed as 1 input to `merge()`.
+
+Additionally, a `src/styles` directory and the input file `site.scss` is passed to `sass()` which will convert your
+`.scss` files into `.css` files, its output (your css) is passed as an input to `autoprefixer()` which will add
+vendor  prefixes (like `-ms` or `-webkit`) to attributes, which is then in turn passed into the `merge()` plugin.
+
+The `merge()` plugin will copy the contents of each of its inputs into its output directory. Thus, it merges our 
+uglified Javascript and out vendor prefixed css. This then becomes our final output and is what is written to 
+our target (destination) directory.
+
+This should all be fairly familiar to you if you've ever written Javascript (or any programming language for that
+matter) before, it's just inputs and output.
+
+## Thinking in Broccoli
+
+There are 3 main concepts to get your head around when using Broccoli:
+
+* [Plugins](#plugins)
+* [Nodes](#nodes)
+* [Trees](#trees)
+
+### Plugins
+
+Plugins are what a build pipeline developer will interact with most. Plugins are what do the actual work of
+transforming input files into output files. The API of a plugin is pretty simple, all that's required is creating
+a class that extends the [broccoli-plugin](https://github.com/broccolijs/broccoli-plugin) base class, and
+implementing a `build()` method, that performs some work or returns a promise.
+
+```js
+const Plugin = require('broccoli-plugin');
+
+class MyPlugin extends Plugin
+{
+  build() {
+    // A plugin can receive single or multiple inputs nodes/directories
+    // The files are available in this.inputPaths[0]/this.inputPaths[1]...
+    // You can do anything with these files that you can do in node
+    // The output of your plugin must write to this.outputPath
+  }
+}
+```
+
+Basic plugins are super simple, anything you can do in node, you can do in the `build()` method. A plugin can
```

Comment:
I think we should avoid saying phrases like `super simple`. Rather lets show how they are "super simple". For example, "a broccoli-plugin has only 1 purpose to transform the files from this.inputPaths directories to their appropriate output in this.outputPath directory when its build function is invoked."